src/nupic/algorithms/Anomaly.cpp

```diff
@@ -109,10 +108,17 @@ Real32 Anomaly::compute(
   {
     score = (score >= binaryThreshold_) ? 1.0 : 0.0;
   }
+  // round result (to avoid near zero rounding errors)
+  score = roundf(score*(1/PRECISION))/PRECISION;
```

Comment:
@breznak Fair enough... I wasn't really aware of your requirements, just trying to help a bit.